systems/plants/shapes/Geometry.cpp

```diff
@@ -59,6 +131,38 @@ namespace DrakeShapes
     return new Capsule(*this);
   }
 
+  void Capsule::getPoints(Matrix4Xd &points) {
+      // Return segment end-points
+      static bool warnOnce = true;
+      if ( warnOnce ) {
+        cerr << "Warning: DrakeShapes::Capsule::getPoints(): This method returns the end points of the line segment that defines the capsule";
+        warnOnce = false;
+      }
```

Comment:
Probably not. I basically took the code (including the warning) from the MATLAB implementation. Will be removed.